coalib/settings/Setting.py

```diff
@@ -144,6 +149,19 @@ def __path_list__(self):
         """
         return [Setting.__path__(elem, self.origin) for elem in self]
 
+    def __url__(self):
```

Comment:
What wha ? Why is here ?

Didnt we decide it should be in `StringConverter` ?
The only reason `__path__` is here is because it needs `self.origin` which doesnt exist in the other thing